nodes/viz/texture_viewer.py

```diff
@@ -369,7 +369,7 @@ def process(self):
             pixels = np.array(self.inputs['Float'].sv_get(deepcopy=False)).flatten()
             width, height = self.texture_width_height
 
-            transfer_to_image(np.resize(pixels, self.total_size), self.texture_name, width, height, mode)
+            transfer_to_image(np.resize(pixels, self.calculate_total_size()), self.texture_name, width, height, mode)
```

Comment:
this allow **rgb** and **rgba** mode but i get some 'red' alert ( i mean the texture viz node became red)